src/models/TechRecordsDAO.ts

```diff
@@ -247,19 +250,13 @@ class TechRecordsDAO {
     };
   }
 
-  public updateVin<T extends Vehicle>(newVehicle: any, oldVehicle: T) {
+  public updateVin<T extends Vehicle>(newVehicle: T, oldVehicle: T) {
     const transactionParams = {
       TransactItems: [
         {
           Put: {
             TableName: this.tableName,
             Item: newVehicle,
-            ConditionExpression:
-              "vin <> :vin AND systemNumber <> :systemNumber",
-            ExpressionAttributeValues: {
-              ":vin": newVehicle.vin,
-              ":systemNumber": newVehicle.systemNumber,
-            },
```

Comment:
Would be nice to have some kind of check here like:
```js
if  (newVehicleRecord.techRecord.some(techRecord => techRecord.statusCode !== "archived")) {
  throw new Error("Has already been updated by someone else")
}
```
but I don't really know if with the deep nesting we have it's really (easily) possible/necessary. Open to comments.